docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,689 @@
+
+[source,java]
+----
+UserTransaction ut = new oracle.tmm.jta.TrmUserTransaction();
+ut.begin(true);
+... // Implement the business logic to begin a transaction.
+----
+
+*Sample script to commit and rollback the transaction:*
+
+To commit a transaction:
+
+[source,java]
+----
+ut.commit();
+----
+
+To rollback a transaction:
+
+[source,java]
+----
+ut.rollback();
+----
+
+==== Building and Deploying the Helidon Participant Application
+
+Obtain the image for MicroTx Free from https://container-registry.oracle.com/[Oracle Container Registry] and add the 'TmmLib' library to the local Maven repository. This library is available in the '<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/' location.
+
+Install the OTMM library to the local Maven repository using the following command:
+
+[source,shell]
+----
+mvn install:install-file -Dfile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-22.3.2.jar -DpomFile=<OTMM_BINARIES_EXTRACTED_PATH>/lib/java/TmmLib-weblogic-22.3.2.pom
+----
+
+Create the Docker image for the Helidon Participant Application and deploy it in the Kubernetes cluster.
+
+===== Creating the Docker Image
+
+In the MicroTx binaries, the Helidon Participant Application samples are available in the `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/helidon-app` location.
+
+Update the `application.yaml` file available in the `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/helidon-app/src/main/resources` location. In this file, specify the database XA connection details of the Helidon Participant Application.
+
+[NOTE]
+====
+In this `application.yaml` file, the local data source, orcl, is referred in the application code. If you want to use a different data source, you should also change the name in the application code.
+====
+
+[source,yaml]
+----
+oracle:
+  ucp:
+    jdbc:
+      PoolXADataSource:
+        localOrcl:
+          URL: jdbc:oracle:thin:@127.0.0.1:1521/orcl
+          connectionFactoryClassName: oracle.jdbc.xa.client.OracleXADataSource
+          user: <db-username>
+          password: <db-password>
+----
+
+Update the `tmm.properties' file available in the `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/helidon-app/src/main/resources` location.
+
+[source,java]
+----
+### oracle.tmm.TcsUrl value the MicroTx coordinator URL
+oracle.tmm.TcsUrl = http(s)://<OTMM_K8S_SERVICE_NAME>:<OTMM_SERVICE_PORT>/api/v1
+oracle.tmm.TcsConnPoolSize = 15
+### oracle.tmm.CallbackUrl refers to the Helidon Teller Application call-back URL along with context path
+oracle.tmm.CallbackUrl = http://<HELIDON_PARTICIPANT_APP_K8s_SERVICE_NAME>:<HELIDON_PARTICIPANT_APP_K8s_SERVICE_PORT>/<HELIDON_PARTICIPANT_APP_CONTEXTPATH>
+##RM Id
+oracle.tmm.xa.<rmid> = HELIDON-TX-RM-ID-FOR-MICROTX-TESTS
+----
+
+Create the Docker image for the Helidon Participant Application, upload the image to the Docker registry, and then deploy the application in the Kubernetes cluster.
```

Comment:
Same comment as above, ie: How do they create the Docker image? Shouldn't they run the docker build command for this on a DockerFile? I looked at the readme.md in `<OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/helidon-app` and it has instruction on how to run the docker build to create the image. The directory also has the Dockerfile that would allow to build the image, so should it not be given as an example instruction? Like:
1. Go to the teller directory:
   ```
   cd <OTMM_BINARIES_EXTRACTED_PATH>/samples/xa/java/weblogic/helidon-app
   ```
2. Run the docker build command to create the image (fill the image_name and tag with the desired values):
    ```
    docker build --network host -t <image_name>:<tag> .
    ```